src/locale/misc/en.yml

```diff
@@ -18,6 +18,8 @@ components:
             noEmpty: This cannot be empty
             edit: Click to edit
             save: Press 'enter' to save
-    ConfirmDialog:
-        defaultTitle: Confirm
-        defaultWarningText: Are you sure you want to do this action? It can not be undone.
+ConfirmDialog:
+    defaultTitle: Confirm
+    defaultWarningText: Are you sure you want to do this action? It can not be undone.
+    button: Confirm
```

Comment:
YAML keys should be sorted alphabetically. When they go through the translation interface, that's the way they come out, so if we don't sort them alphabetically to begin with we will have to deal with unnecessarily big diffs when we apply translations.